### PR TITLE
Iv 2436 coreos support

### PR DESCRIPTION
**Pull request by lopaka**
Replaced rll/collectd.sh with rll/enable-monitoring.sh (RL10 Linux Enable Monitoring) to do RL monitoring for CoreOS and collecd installation for others.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ Changelog for the RightLink10 Base ServerTemplate
 Unreleased Changes
 ------------------
 - Update scripts with support for CoreOS
+- Replaced rll/collectd.sh with rll/enable-monitoring.sh (RL10 Linux Enable Monitoring)
```
Comment (psschroeter) [inline CHANGES.md:7]: How about the entires in metadata.rb?

Comment (psschroeter) [inline CHANGES.md:7]: Bump the version to avoid overwriting 10.2.1 too I guess.

Comment (psschroeter) [inline CHANGES.md:7]: The readme also mentions collectd.sh

Comment (lopaka) [inline CHANGES.md:7]: All 3 items addressed

**Pull request merged**